modules/099-ceph-csi/crds/cephcsi.yaml

```diff
@@ -0,0 +1,141 @@
+                          namePostfix:
+                            description: |
+                              Part of the StorageClass name after `-`. The name from the CustomResource CephCSIDriver is used as the first part.
+                            type: string
+                          pool:
+                            description: |
+                              Pool name for creating RBD images.
+                            type: string
+                          reclaimPolicy:
+                            description: |
+                              The reclaim policy for a PersistentVolume.
+                              More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming
+                            type: string
+                            enum:
+                              - Delete
+                              - Retain
+                            default: Retain
+                          allowVolumeExpansion:
+                            description: |
+                              Allows the users to resize the volume by editing the corresponding PVC object.
+                            type: boolean
+                            default: true
+                          mountOptions:
+                            description: |
+                              List of mount options.
+                            type: array
+                            items:
+                              type: string
+                          defaultFSType:
+                            description: |
+                              Default fstype.
+                            type: string
+                            enum:
+                              - ext4
+                              - xfs
+                            default: ext4
+                cephfs:
+                  type: object
+                  properties:
+                    subvolumeGroup:
+                      description: |
+                        CephFS subvolume group name.
+                      type: string
+                    storageClasses:
+                      type: array
+                      items:
+                        type: object
+                        required:
+                          - namePostfix
+                        properties:
+                          namePostfix:
+                            description: |
+                              Part of the StorageClass name after `-`. The name from the CustomResource CephCSIDriver is used as the first part.
+                            type: string
+                          pool:
+                            description: |
+                              Pool name for creating RBD images.
+                            type: string
+                          reclaimPolicy:
+                            description: |
+                              The reclaim policy for a PersistentVolume.
+                            type: string
```

Comment:
I assume it should be the same schema as for the rbd section:
```
enum:
- Delete
- Retain
default: Retain
```

Reply:
Fixed